pkg/bootstrap/platform/k8s.go

```diff
@@ -156,7 +156,7 @@ spec:
             secretName: cl-dataplane
       containers:
         - name: dataplane
-          {{ if (eq .dataplaneType .dataplaneTypeEnvoy) }}image: cl-dataplane{{ else }}image: cl-go-dataplane{{ end }}
+          {{ if (eq .dataplaneType .dataplaneTypeEnvoy) }}image: {{.containerRegistry}}cl-dataplane{{ else }}image: {{.containerRegistry}}cl-go-dataplane{{ end }}
```

Comment:
```suggestion
          image: {{.containerRegistry}}{{ if .containerRegistry }}/{{ end }}{{ if (eq .dataplaneType .dataplaneTypeEnvoy) }}cl-dataplane{{ else }}cl-go-dataplane{{ end }}
```

Reply:
I prefer to do it in the go code, see my next patch.